docs/topics/strings.md

```diff
@@ -12,9 +12,9 @@ You can iterate over these characters with a `for` loop:
 
 ```kotlin
 fun main() {
-val str = "abcd"
-//sampleStart
-for (c in str) {
+    val str = "abcd"
+    //sampleStart
+    for (c in str) {
```

Comment:
This is a strange indentation change. Probably, playground will reformat it anyway, but would be nice to have it consistent here as well.

Reply:
I formatted this and the rest of the code snippets in IntelliJ. Please let me know if that solves this issue or is more left to do.

Reply:
@ilya-g what's strange about the indentation? 4 spaces are what we should use for indentation according to our coding conventions 🤔 It was a bug that we didn't have these indentations in all the samples before.

Reply:
The contents of the for loop are misaligned

Reply:
Were misaligned in the first commit. Note it's ok.